Header header
float32 speed
float32 turn_center_x
float32 turn_center_y
float32 strafing_angle
bool    is_strafing
bool    is_braking

